robyn/__init__.py

```diff
@@ -140,7 +146,8 @@ def start(self, url: str = "127.0.0.1", port: int = 8080):
             self.directories,
             self.request_headers,
             self.router.get_routes(),
-            self.middleware_router.get_routes(),
+            self.middleware_router.get_global_middlewares(),
+            self.middleware_router.get_route_middlewares(),
```

Comment:
@AntoineRR , this is a nice naming change! :D

Reply:
Thanks, I think it's best to have `route_middlewares` and `global_middlewares`, it makes more sense to me :smile: